components/TwitterTestimonials.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import Tweet from "./tweets";
+
+const TwitterTestimonials = () => {
+  const tweets = [
```

Comment:
Since this is constant you should define it outside the component so that it would not reinitialize on every render

Reply:
fixed it!